lenskit-api/src/main/java/org/lenskit/api/ItemBasedItemRecommender.java

```diff
@@ -29,62 +26,68 @@
 
 
 /**
- * The interface for recommendation based on the items only. The difference from {@link
- * ItemRecommender} is that the input is only the item or list of items instead of user specific
- * information. This interface can be used to provide a Find Similar Items / People Also Liked
- * feature.
+ * Recommends items that go with a set of reference items. This interface is distinguished from
+ * {@link ItemRecommender} in that it uses a set of reference items instead
+ * of a user as the basis for computing scores. This component can be used to provide a Find Similar Items
+ * or People Also Liked feature.
```

Comment:
Not 100% sure, (since this note applies to all algorithms) that it depends on which algorithm/configuration whether it is reasonable to use this for similar items. Example:

consider recommender on amazon which is fed in a mid-range microwave. Similar items are other microwaves, good basket recommendations are probably toasters and other kitchenware. Both would use this API, but the meanings are different.